src/test/kotlin/org/jitsi/rtp/rtcp/rtcpfb/payload_specific_fb/RtcpFbRembPacketTest.kt

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.rtp.rtcp.rtcpfb.payload_specific_fb
+
+import io.kotlintest.IsolationMode
+import io.kotlintest.shouldBe
+import io.kotlintest.specs.ShouldSpec
+import org.jitsi.rtp.rtcp.RtcpHeaderBuilder
+
+internal class RtcpFbRembPacketTest : ShouldSpec() {
```

Comment:
Do we want to have a test of parsing a verbatim REMB packet (captured from wireshark or whatever)?

Reply:
I've added a couple of tests with the same packets that were used to test the wireshark dissector.